src/app/utils/MGPFallible.ts

```diff
@@ -86,6 +91,9 @@ class MGPFallibleFailure<T> extends MGPFallible<T> {
     public getReasonOr(_value: string): string {
         return this.getReason();
     }
+    public and(_other: MGPFallible<T>): MGPFallible<T> {
```

Comment:
p-e juste other, why _other (same for _value plus haut)

Reply:
c'est pour faire plaisir à mon linter qui se plaint qu'il est pas utilisé, ça permet de dire « je sais que j'utilise pas cet argument, c'est voulu »

Reply:
dans tous les cas c't'obsolève c'machin la!